File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/utils/UnitConverter.java
package de.limbusdev.guardianmonsters.utils;

import de.limbusdev.guardianmonsters.Constant;

public class UnitConverter
{
    public static int pixelsToMeters(int pixels) {
        return pixels/ Constant.PIXELS_PER_METER;
    }

    public static int metersToPixels(int meters) {
        return meters * Constant.PIXELS_PER_METER;
    }

    public static int tilesToPixels(int tiles) {
        return tiles * Constant.TILE_SIZE;
    }

    public static int pixelsToTiles(int tiles) {
        return Constant.TILE_SIZE/tiles;
    }
}

File: GuardianMonsters/core/src/de/limbusdev/guardianmonsters/utils/UnitConverter.kt
package de.limbusdev.guardianmonsters.utils

import de.limbusdev.guardianmonsters.Constant

object UnitConverter
{
    fun pixelsToMeters(pixels: Int): Int =  pixels / Constant.PIXELS_PER_METER

    fun metersToPixels(meters: Int): Int = meters * Constant.PIXELS_PER_METER

    fun tilesToPixels(tiles: Int): Int = tiles * Constant.TILE_SIZE

    fun pixelsToTiles(tiles: Int): Int = Constant.TILE_SIZE / tiles
}
